src/css/styles/_typography.scss

```diff
@@ -26,54 +26,33 @@
   font-weight: normal;
 }
 
-.c-link--nav,
-.c-link--nav-mobile,
+.c-menu__link,
+.c-menu--mobile__link,
 .c-lang-dd__option-link,
-.banner-link,
-.h-header,
+.c-lang--mobile__item,
+.c-btn,
+.c-section__header,
+.c-cover__title,
+.c-quote__header,
+.c-quote__text,
+.c-quote__author,
+.c-quote__author-position,
+.c-title,
 .c-list__title,
-.c-contact__name-txt,
-.c-btns__btn {
+.c-contact__name-text,
+.h-btn-divider {
   font-family: "Praktika Bold", "Source Sans Pro", "Lucida Grande", sans-serif;
 }
 
 .app,
-.c-link--nav-mobile-kontakt,
-.h-txt-thin {
+.c-menu--mobile__contact-link {
   font-family: "Praktika Medium", "Source Sans Pro", "Lucida Grande", sans-serif;
 }
```

Comment:
Maybe we could move `"Praktika Medium", "Source Sans Pro", "Lucida Grande", sans-serif;` and `"Praktika Bold", "Source Sans Pro", "Lucida Grande", sans-serif;  to variables.

We should use default font in `app`:

```scss
.app {
  font-family: $font-praktika-medium;
}
```

and

also for each component that should have different font (in example `c-list__title`) you should specify the font explicitly